### PR TITLE
[tests] add test to make sure BR works in FED role

### PR DESCRIPTION
**Pull request by simonlingoogle**

### CODE REVIEW

File: tests/scripts/thread-cert/border_router/test_border_router_as_fed.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        self.assertEqual('child', br.get_state())
+
+        self.simulator.go(10)
+        self.assertEqual('child', br.get_state())
+
+        # Leader can ping to/from the Host on infra link.
+        self.assertTrue(leader.ping(host.get_ip6_address(config.ADDRESS_TYPE.ONLINK_ULA)[0]))
+        self.assertTrue(host.ping(leader.get_ip6_address(config.ADDRESS_TYPE.OMR)[0], backbone=True))
+
+        self.simulator.go(5)
```
Comment (abtink) [inline tests/scripts/thread-cert/border_router/test_border_router_as_fed.py:97]: I think we can remove this last `go(5)`?

**Review by abtink — APPROVED** [review]
LGTM. 👍

File: tests/scripts/thread-cert/border_router/test_border_router_as_fed.py (hunk 1 above)
Comment (simonlingoogle) [reply tests/scripts/thread-cert/border_router/test_border_router_as_fed.py:97]: We tend to add a small delay at the end to make sure PCAP contains all traffic.

Comment (abtink) [reply tests/scripts/thread-cert/border_router/test_border_router_as_fed.py:97]: Good to know. Thanks.

**Review by jwhui — APPROVED** [review]

**Pull request merged**